lib/vagrant-openshift/action/run_openshift_tests.rb

```diff
@@ -92,6 +92,12 @@ def call(env)
           cmd = cmd_env.join(' ') + ' ' + build_targets.join(' ')
           env[:test_exit_code] = run_tests(env, [cmd], true)
 
+          if env[:test_exit_code] == 0 && @options[:extended_test_packages].length > 0
+            extended_cmd = 'hack/test-extended.sh ' + @options[:extended_test_packages]
+            env[:test_exit_code] = run_tests(env, [extended_cmd], true)
```

Comment:
@danmcp this takes args from test_openshift.rb and makes them part of a shell command. I don't know all the ways this can be invoked... are there any where we wouldn't trust user input?